script/job_generator/generate_job_yaml.py

```diff
@@ -449,7 +475,9 @@ def create_job(
             job_name + "-" + job[HOST_COMPILER][NAME] + job[HOST_COMPILER][VERSION]
         )
     # if Clang-CUDA is the device compiler, add also the CUDA SDK version to the name
-    if job[DEVICE_COMPILER][NAME] == CLANG_CUDA:
+    if (job[DEVICE_COMPILER][NAME] == CLANG_CUDA
+        and ALPAKA_ACC_GPU_CUDA_ENABLE in job
+        and job[ALPAKA_ACC_GPU_CUDA_ENABLE][VERSION] != OFF_VER):
```

Comment:
Apparently there is a bug somewhere in the job generator which leads to `clang-cuda` jobs being generated without ÀLPAKA_ACC_GPU_CUDA_ENABLE` being set. This leads to a `KeyError` here during the `generate` stage.